src/constants/geoChart.ts

```diff
@@ -0,0 +1,30 @@
+// (C) 2019 GoodData Corporation
+
+export const DEFAULT_PUSHPIN_LOCATION = {
+    LAT: 34,
+    LNG: 5,
+    ZOOM: 2,
+};
```

Comment:
Please refer to this [comment](https://github.com/gooddata/gooddata-react-components/pull/1307/#discussion_r362358303)